rust/src/lib.rs

```diff
@@ -2762,4 +2797,66 @@ mod tests {
 
         assert_eq!(map2.keys(), AssetNames(vec![name33, name11, name22]));
     }
+
+    #[test]
+    fn mint_to_value() {
+        let policy_id1 = PolicyID::from([0u8; 28]);
+        let policy_id2 = PolicyID::from([1u8; 28]);
+        let name1 = AssetName::new(vec![0u8, 1, 2, 3]).unwrap();
+        let name2 = AssetName::new(vec![0u8, 4, 5, 6]).unwrap();
+        let amount1 = BigNum::from_str("1234").unwrap();
+        let amount2 = BigNum::from_str("5678").unwrap();
+
+        let mut mass1 = MintAssets::new();
+        mass1.insert(&name1, Int::new(&amount1));
+        mass1.insert(&name2, Int::new(&amount2));
+
+        let mut mass2 = MintAssets::new();
+        mass2.insert(&name1, Int::new(&amount2));
+        mass2.insert(&name2, Int::new(&amount1));
+
+        let mut mint = Mint::new();
+        mint.insert(&policy_id1, &mass1);
+        mint.insert(&policy_id2, &mass2);
+
+        let val = mint.to_value().unwrap();
+
+        assert_eq!(val.coin(), Coin::zero());
+        assert!(val.multiasset().is_some());
```

Comment:
Ah, I just had some problems before with debugging which part of the test is failing exactly because the message on just `.unwrap` was pointing to the `Result` or `Option` sources, where the error is thrown, instead of pointing to the place in the test where it fails. So the test is failing alright, but to figure out which unwrap exactly is causing the problem might be problematic.

Example:

![image](https://user-images.githubusercontent.com/5585355/141435404-da81edd6-1394-4c08-aa18-d2bfd88c11cb.png)

vs

![image](https://user-images.githubusercontent.com/5585355/141435447-697c84a5-efa3-4805-954e-c7003fbe2106.png)

When some test cases have 20 unwraps in them - that turnes into a problem.